### PR TITLE
[REVIEW] Added Ampere support

### LINKED ISSUE #197: [FEA] Added Ampere support
In order to add Ampere support we need to add support for CUDA 11 and CC 8.0

### PR DESCRIPTION
**Pull request by mnicely**
This PR add compute capability 8.0 to the build process and updates all fatbins.

This closes #197

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by mnicely** [comment]
rerun tests

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## New Features
 - PR #185 - Add function to translate PyCUDA gpuarray to CuPy ndarray
 - PR #195 - Add explicit FIR filter functionality
+- PR #197 - Added Ampere support
```
Comment (BradReesWork) [inline CHANGELOG.md:6]: PR number in file does not match the actual PR number

File: build.sh [hunk 2]
```diff
@@ -110,7 +110,7 @@ fi
 # Build fatbins
 SRC="cpp/src"
 FAT="python/cusignal"
-FLAGS="-std=c++11"
+FLAGS="-std=c++17"
```
Comment (BradReesWork) [inline build.sh:113]: have you seen any issues with setting the c++17 flag?

Comment (mnicely) [reply build.sh:113]: No issues for our stuff.
I'm using `if constexpr` to minimize runtime branching when selecting device based on type. It requires C++17.

File: CHANGELOG.md (hunk 1 above)
Comment (mnicely) [reply CHANGELOG.md]: Thanks! That fixed the CI issue I believe.

**Comment by mnicely** [comment]
@BradReesWork Let me know what you think about the changes.

**Review by awthomp — APPROVED** [review]

**Review by BradReesWork — APPROVED** [review]

**Pull request merged**